packages/vscode-extension/src/debug/deleteAadHelper.ts

```diff
@@ -0,0 +1,120 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { dotenvUtil } from "@microsoft/teamsfx-core/build/component/utils/envUtil";
+import M365TokenInstance from "../commonlib/m365Login";
+import * as globalVariables from "../globalVariables";
+import * as fs from "fs-extra";
+import * as path from "path";
+import { GraphScopes } from "@microsoft/teamsfx-core";
+import axios from "axios";
+import { ConvertTokenToJson } from "../commonlib/codeFlowLogin";
+import VsCodeLogInstance from "../commonlib/log";
+import * as util from "util";
+import { localize } from "../utils/localizeUtils";
+import { ExtTelemetry } from "../telemetry/extTelemetry";
+import { TelemetryEvent } from "../telemetry/extTelemetryEvents";
+import { FxError } from "@microsoft/teamsfx-api";
+
+const defaultNotificationLocalFile = ".notification.localstore.json";
+export async function deleteAad() {
+  try {
+    if (globalVariables.deleteAadInProgress) {
+      return;
+    }
+    globalVariables.setDeleteAadInProgress(true);
+    const projectPath = globalVariables.workspaceUri!.fsPath;
+    const envFile = path.resolve(projectPath, "env", ".env.local");
+    const userFile = path.resolve(projectPath, "env", ".env.local.user");
+    if (!fs.existsSync(envFile) || !fs.existsSync(userFile)) {
+      return;
+    }
+    const envData = dotenvUtil.deserialize(fs.readFileSync(envFile, "utf-8"));
+    const userEnvData = dotenvUtil.deserialize(fs.readFileSync(userFile, "utf-8"));
+    if (!envData.obj["BOT_ID"] && !envData.obj["AAD_APP_CLIENT_ID"]) {
```

Comment:
User can customize the outputted environment variable name in teamsapp.yml, we may also read the actual environment variable name instead of hard code